lib/quibble.mjs

```diff
@@ -6,7 +6,9 @@ export const ignoreCallsFromThisFile = quibble.ignoreCallsFromThisFile
 export const config = quibble.config
 export const isLoaderLoaded = quibble.isLoaderLoaded
 
-global.__quibble = { stubModuleGeneration: 1 }
+export const globalPreload = () => {
```

Comment:
This won't work in Node 14 because Node 14 uses a different function name [`getGlobalPreloadCode`](https://nodejs.org/docs/latest-v14.x/api/esm.html#esm_getglobalpreloadcode).

Also, while you're at it: these functions should be returning a `string`, and you're returning `undefined` here. In the interest of conforming to the spec, I suggest you return an empty string.

Reply:
Thanks for the review @giltayar! I have added the Node 14 hook, used normal function declaration to be in line with the other code, set the global state in user-land through the string (does not matter much where this happens I think).